suite-native/app/src/App.tsx

```diff
@@ -59,11 +62,12 @@ const AppComponent = () => {
 
     return (
         <FormatterProvider config={formattersConfig}>
-            <MessageSystemBannerRenderer />
+            {isOnboardingFinished && <OfflineHeader />}
+            {isOnboardingFinished && <MessageSystemBannerRenderer />}
```

Comment:
I believe we should be able to show message system banner even before onboarding is finished.

Reply:
22ed29dd69